packages/web-components/src/dialog-body/dialog-body.ts

```diff
@@ -8,10 +9,15 @@ import { attr, FASTElement } from '@microsoft/fast-element';
  * @extends FASTElement
  */
 export class DialogBody extends FASTElement {
-  /**
-   * @public
-   * Indicates whether the dialog has a title action
-   */
-  @attr({ mode: 'boolean', attribute: 'no-title-action' })
-  public noTitleAction: boolean = false;
+  public clickHandler(event: MouseEvent): boolean | void {
```

Comment:
nit: the function should be documented and include a release tag (probably `@internal`)